var/spack/repos/builtin/packages/aml/test/hello_world.c

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+* Copyright 2019 UChicago Argonne, LLC.
+* (c.f. AUTHORS, LICENSE)
+*
+* This file is part of the AML project.
+* For more info, see https://xgitlab.cels.anl.gov/argo/aml
+*
+* SPDX-License-Identifier: BSD-3-Clause
+******************************************************************************/
```

Comment:
It would be preferable to not have to deal with potential licensing issues.  I don't see this file when I stage the sources.